packages/react-core/src/demos/Toolbar.md

```diff
@@ -19,556 +19,6 @@ import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
-              <MenuToggle ref={toggleRef} isExpanded={downloadExpanded} onClick={this.onDownloadToggle}>
-                Download
-              </MenuToggle>
-            )}
-          >
-            <DropdownList>{downloadDropdownItems}</DropdownList>
-          </Dropdown>
-        </ToolbarItem>
-      </React.Fragment>
-    );
-
-    const rightAlignedItemsMobile = (
-      <React.Fragment>
-        <ToolbarItem visibility={{ default: 'visible', '2xl': 'hidden' }}>
-          <Tooltip position="top" content={<div>External logs</div>}>
-            <Dropdown
-              isOpen={externalExpandedMobile}
-              onOpenChange={(isOpen) => this.setState({ externalExpandedMobile: isOpen })}
-              onSelect={this.onExternalSelectMobile}
-              toggle={(toggleRef) => (
-                <MenuToggle
-                  ref={toggleRef}
-                  isExpanded={externalExpandedMobile}
-                  onClick={this.onExternalToggleMobile}
-                  icon={<ExternalLinkAltIcon />}
-                  aria-label="External logs"
-                />
-              )}
-            >
-              <DropdownList>{externalDropdownItems}</DropdownList>
-            </Dropdown>
-          </Tooltip>
-        </ToolbarItem>
-        <ToolbarItem visibility={{ default: 'visible', '2xl': 'hidden' }}>
-          <Tooltip position="top" content={<div>Download</div>}>
-            <Dropdown
-              isOpen={downloadExpandedMobile}
-              onOpenChange={(isOpen) => this.setState({ downloadExpandedMobile: isOpen })}
-              onSelect={this.onDownloadSelectMobile}
-              popperProps={{ position: 'right' }}
-              toggle={(toggleRef) => (
-                <MenuToggle
-                  ref={toggleRef}
-                  isExpanded={downloadExpandedMobile}
-                  onClick={this.onDownloadToggleMobile}
-                  icon={<DownloadIcon />}
-                  aria-label="Download"
-                />
-              )}
-            >
-              <DropdownList>{downloadDropdownItems}</DropdownList>
-            </Dropdown>
-          </Tooltip>
-        </ToolbarItem>
-      </React.Fragment>
-    );
-
-    const rightAlignedItems = (
-      <React.Fragment>
-        <ToolbarItem>{LogsSearchInput}</ToolbarItem>
-        {rightAlignedItemsDesktop}
-        {rightAlignedItemsMobile}
-        <ToolbarItem>
-          <Tooltip position="top" content={<div>Expand</div>}>
-            <Button variant="plain" aria-label="expand">
-              <ExpandIcon />
-            </Button>
-          </Tooltip>
-        </ToolbarItem>
-      </React.Fragment>
-    );
-
-    const items = (
-      <React.Fragment>
-        <ToolbarGroup align={{ default: 'alignLeft' }}>{leftAlignedItems}</ToolbarGroup>
-        <ToolbarGroup align={{ default: 'alignRight' }}>{rightAlignedItems}</ToolbarGroup>
-      </React.Fragment>
-    );
+```js file="./examples/Toolbar/ConsoleLogViewerToolbar.tsx" isFullscreen
```

Comment:
Along with many of our existing PF demos which import `DashboardWrapper`, [see BackToTop.md for example],(https://github.com/patternfly/patternfly-react/blob/1c70d6855fc5506bc2d683db91443219fa911189/packages/react-core/src/demos/BackToTop.md?plain=1#L14) this will have to remain as a js filetype for now until the work for #9544 is complete. Otherwise the DashboardWrapper import breaks the CI build.